### ISSUE TITLE
Some visual samples

### ISSUE THREAD

**Issue by HosseinMA96** [daanvdh/JavaDataFlow#3]
Hi,

Is there any visual sample of the actual graph (nodes and edges) for an exemplary program?

Thanks.

**Comment by daanvdh** [comment]
Currently there is no way to output the DFG as a picture. I made an effort to have the toString methods human readable, hopefully that helps. If someone makes an effort in converting the DFG to a picture I'll be more than happy to assist in getting it to work.

**Comment by zhizhixiongxuwei** [comment]
So your dafaflow is similar with the concept Control Flow Graph(CFG) or Call Graph(CG)??

**Comment by daanvdh** [comment]
Yes, calls to methods from within the same class are present in the DFG.
